scopes/react/component-highlighter/component-highlighter/component-highlighter.module.scss

```diff
@@ -0,0 +1,11 @@
+.highlighter {
+  border: solid var(--border-color, #ededed);
+  border-width: 0px;
+  box-sizing: border-box;
+  transition: all 300ms;
+  height: 100vh;
```

Comment:
100vh? why not 100%?

Reply:
html/body is not 100%, I height 100% would not have an effect